pkg/rpc/context.go

```diff
@@ -1050,18 +1051,32 @@ type delayingHeader struct {
 	DelayMS  int32
 }
 
+func makeDialCtx(
+	dialCtx context.Context, remoteNodeID roachpb.NodeID, target string, class ConnectionClass,
+) context.Context {
+	if remoteNodeID == 0 {
+		dialCtx = logtags.AddTag(dialCtx, "rnode?", nil)
+	} else {
+		dialCtx = logtags.AddTag(dialCtx, "rnode", remoteNodeID)
+	}
+	dialCtx = logtags.AddTag(dialCtx, "raddr", target)
```

Comment:
This gets marked as sensitive and redacted in logs. Is that intentional? I'm not sure if we consider IP addresses to be sensitive, but it would make debugging a bit harder.